Misc/holorepo_demo/README.md

```diff
@@ -0,0 +1,77 @@
+# HoloRepository Demo
+This folder contains demo data to setup HoloRepository and the README will document the steps to spin up the necessary infrastructure for HoloRepository
```

Comment:
Maybe the name of this sub-dir could be more expressive. Maybe `project_setup`? `setup_infrastructure`? Or be fun (i think you had this idea with the branch name) and go `holo_world`

Reply:
ok holo_world it is